src/Runner.Worker/ActionManager.cs

```diff
@@ -395,6 +395,10 @@ public Definition LoadAction(IExecutionContext executionContext, Pipelines.Actio
                             Trace.Info($"Action cleanup plugin: {plugin.PluginTypeName}.");
                         }
                     }
+                    else if (definition.Data.Execution.ExecutionType == ActionExecutionType.Composite && !String.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
+                    {
+                        // Don't do anything for now
```

Comment:
check what we do for nodejs, line 373, just dump all steps into log might be good.

Reply:
we can just dump the step as JSON in log.

Reply:
good idea - thanks. 

(Note to self: do the same for when I add env flow to composite actions)